wiki/Tournaments/GBC/GBC_2022_Spring/en.md

```diff
@@ -0,0 +1,201 @@
+
+#### Win conditions
+
+- In all stages, teams need to win 6 maps to win a match (best of 11).
+
+### Match instructions
+
+1. A referee will create a multiplayer room 15 minutes in advance. Players must gather during this period.
+   - Room settings are osu!mania, Team VS, Win Condition: Score v2. Room name must be `GBC2022Spring: ({TeamRed}) vs ({TeamBlue})`.
+   - The team mentioned first in the room name must be the red team, and the team mentioned second in the room name must be the blue team.
+2. Each captain can ban **one beatmap** from the pool. These beatmaps cannot be picked by any team during the entire match.
+3. Beatmap selection will alternate between each captain selecting a beatmap out of the mappool.
+4. Each captain must use `!roll` once in `#multiplayer`.
+   - The winner of the `!roll` starts picking the first beatmap of the match.
+   - The loser of the `!roll` starts banning one beatmap, followed by the winner of the `!roll` to ban a beatmap. 
+5. After each pick, the captain may ban one player of the opposite player from playing that beatmap in the match, except in a tiebreaker.
+   - Any player can only be banned like this **once** per match.
+   - Banning players is optional.
+6. Teams will have 2 minutes to pick or ban a beatmap, 2 minutes to ban an opposing player, and 3 minutes to get ready. If a team takes more time than allotted, the procedures adopted will be as follows:
+   - For the first occurrence:
+     - The team will receive a verbal warning from the referee
+   - On subsequent occurrences:
+     - For bans of opposing players, the captain forfeits the ban.
+     - For map bans, the captain forfeits the ban.
+     - For map picks, a random map will be chosen using the `!roll` command from the remaining mappool, excluding the tiebreaker map.
+     - For a ready timer, the referee will issue the `!mp start 10` command, regardless of how many players from each team are present in the lobby, using `!mp kick` on any extra players for each team, starting from the top (i.e. the first valid player combination for each team will be forced to play the pick). The results for such games are to be taken as is.
+   - Repeat offenders may receive further sanctions from the host.
+   - Each team will receive **one** "tactical timeout" of **two minutes**, to be used as extra time to pick or ban a beatmap. *The tactical timeout is optional.*
+7. The referee reserves the right to modify these rules or decide if a team has violated these rules in matches they are responsible for, if absolutely necessary.
+
+### Mappool instructions
+
+1. There will be one mappool for each stage.
+   - The whole Regular stage will be seen as a single stage, meaning the same mappool will be used in all matches in the Regular stage.
+2. Each mappool consists of a fixed amount of maps each stage which will all be played under FreeMod conditions. This means that there is a unique FreeMod bracket.
+3. All mappool contains 13 beatmaps.
```

Comment:
woops

```suggestion
3. All mappools contain 13 beatmaps.
```

Reply:
hah i'm wondering it's some special usage i don't know.

Reply:
"mappool" was supposed to be plural ("all mappools")

"contain" would only be "contains" if whatever before it was in singular